build.gradle

```diff
@@ -178,6 +178,7 @@ dependencies {
     implementation "org.apache.lucene:lucene-queries:$luceneVersion"
     implementation "org.apache.lucene:lucene-analysis-common:$luceneVersion"
     implementation "org.apache.lucene:lucene-highlighter:$luceneVersion"
+    implementation 'org.apache.commons:commons-text:1.13.0'
```

Comment:
No need for Apache commons text, we already have a Levenstein distance implementation
https://github.com/JabRef/jabref/blob/c6e317dbadb01ca56002d8dadf41ee35962a328b/src/main/java/org/jabref/logic/util/strings/StringSimilarity.java#L8